pkg/sdk/logging/model/syslogng/output/elasticsearch_datastream.go

```diff
@@ -0,0 +1,59 @@
+// Copyright © 2024 Kube logging authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package output
+
+// +name:"Elasticsearch datastream"
+// +weight:"200"
+type _hugoElasticsearchDatastream interface{} //nolint:deadcode,unused
+
+// +docName:"Sending messages over Elasticsearch datastreams"
+/*
+Based on the [ElasticSearch datastream destination of AxoSyslog core](https://axoflow.com/docs/axosyslog-core/chapter-destinations/configuring-destinations-elasticsearch-datastream/).
+
+## Example
+
+{{< highlight yaml >}}
+apiVersion: logging.banzaicloud.io/v1beta1
+kind: SyslogNGOutput
+metadata:
+  name: elasticsearchdatastream
+spec:
+  elasticsearchdatastream:
```

Comment:
isn't it `elasticsearch-datastream`?
```suggestion
  elasticsearchdatastream:
```

Reply:
Yup, thanks!